.github/workflows/staging-build-deploy.yml

```
@@ -0,0 +1,63 @@
+          ./gradlew clean build -p aics-admin -x test
+          ./gradlew clean build -p aics-api -x test
+          ./gradlew clean build -p aics-auth -x test
+
+      - name: Build Docker images
+        run: |
+          docker build -t aics-admin:${GITHUB_SHA} -f aics-admin/Dockerfile aics-admin
+          docker build -t aics-api:${GITHUB_SHA} -f aics-api/Dockerfile aics-api
+          docker build -t aics-auth:${GITHUB_SHA} -f aics-auth/Dockerfile aics-auth
+
+      - name: Save Docker images as tar
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:2:24: Double quote to prevent globbing and word splitting

(shellcheck)